packages/application-shell/src/components/inject-reducers/inject-reducers.js

```diff
@@ -24,7 +24,7 @@ export class InjectReducers extends React.PureComponent {
     this.setState({ areReducersInjected: true });
   }
 
-  UNSAFE_componentWillUnmount() {
+  componentWillUnmount() {
```

Comment:
`componentWillUnmount` is not a deprecated lifecycle method. I guess this was confused with `componentWillMount`

Reply:
Prolly a find and replace gone rogue.